src/sagemaker/session.py

```diff
@@ -43,6 +43,8 @@
 
 LOGGER = logging.getLogger("sagemaker")
 
+RESOURCE_METADATA_FILE = "/opt/ml/metadata/resource-metadata.json"
```

Comment:
is this specific to notebook instances? if so, I might rename this to `NOTEBOOK_METADATA_FILE` just to make it clear where we're expecting this file to exist

Reply:
Yes, [that file](https://docs.aws.amazon.com/sagemaker/latest/dg/nbi-metadata.html) is specific to the notebook instance. Good idea - I've renamed RESOURCE_METADATA_FILE to NOTEBOOK_METADATA_FILE to clarify this.